doc/frametype.rst

```diff
@@ -0,0 +1,59 @@
+.. highlight:: rest
+
+.. _frame_types:
+
+**********
+Frame Type
+**********
+
+.. index:: Frame_Type
+
+Overview
+========
+
+Every raw data file ingested by PYPIT is automatically
+assigned one or more frametype values.  This is to separate
+calibration files, science frames, etc.  The assignments
+are guided by criteria given in the default settings file
+for each spectrograph (e.g. setttings.kast_blue).  One
+should not modify the default files but if you have a
+suggestion for improvement consult with the PYPIT authors.
+
+
+Definitions
+===========
+
+Here are the frametype values adopted in PYPIT:
+
+========= =============================================================
+Frametype Description
+========= =============================================================
+arc       Spectrum of one or more calibration arc lamps
+bias      Bias frame;  typically a 0s exposure with the shutter closed
+pixelflat Spectrum taken to correct for pixel-to-pixel detector variations
+          Often an exposure using a flat lamp, but
+          for observations in the very blue, this may be on-sky
+slitflat  Spectrum taken to define the slit edges and correct for
+          illumination variations across the slit
+trace     ???
```

Comment:
trace -> pinhole

How about this for a description:

Spectrum taken through a pinhole slit (i.e. a very short slit length), and is used to define the centre if a slit (currently, this frame is only used for echelle data reduction). Often this is an exposure using a flat lamp, but one can in principle use a standard star frame too (or a science frame if the spectrum is uniform).